Modified the HMIP-PSM code to support Voltage and Current

gooooba opened the pull request.

twendt commented:
There is still a merge conflict in app.json. You can remove the change since I now use compose to create the app.json.

gooooba commented:
Hi,

 

Tried to resolve the conflict but I don't get it... I modified suggested changes, copied your app.json, deleted app.json,  but just not working...

If you can point me to what needs to be changed?

 

Br,

Dusan

 

From: twendt <notifications@github.com> 
Sent: Tuesday, March 5, 2019 8:03 AM
To: twendt/homey-matic <homey-matic@noreply.github.com>
Cc: gooooba <dd@vklop.com>; Author <author@noreply.github.com>
Subject: Re: [twendt/homey-matic] Modified the HMIP-PSM code to support Voltage and Current (#3)

 

There is still a merge conflict in app.json. You can remove the change since I now use compose to create the app.json.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/twendt/homey-matic/pull/3#issuecomment-469564366> , or mute the thread <https://github.com/notifications/unsubscribe-auth/ADKwoJ5ULdzSHAucUYsQdB1JL9TeWsSvks5vTha0gaJpZM4bZ0p1> .

twendt commented:
I manually added your code and commit it. Therefore I will close this PR.
